cli/__snapshots__/install_spec.js

```diff
@@ -9,7 +9,8 @@ Skipping installation:
 `
 
 exports['skip installation 1'] = `
-Skipping binary installation. Env var 'CYPRESS_SKIP_BINARY_INSTALL' was found.
+⚠ Warning: Skipping binary installation. Env var CYPRESS_INSTALL_BINARY was 0.
```

Comment:
Sometimes you print 'env var' and other times you print 'environment variable' - I'd prefer if all errors say 'environment variable'